christfellowshipchurch/web-app-v2
# Fix add remove content resources

Fixes adding and removing content items and hides resource items that do not have titles.

TO TEST:
Try adding and removing a content item to a groups resources. Also double check to make sure adding and removing a url still works.

![2021-05-06 10 04 10](https://user-images.githubusercontent.com/2528817/117321281-9da01f00-ae52-11eb-8d32-b5cee53e6d8c.gif)
